minishift/replication_controller.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ReplicationController
```

Comment:
How about `Deployment`? we'll get same functionality of RC, along with rollout/rollback using `ReplicaSet`?

Reply:
Done in https://github.com/redhat-developer/git-service/pull/1/commits/b11eebe97c974ff98c10e4b8a7f97c3f5d82021f